contracts/payloads/IGP95/PayloadIGP95.sol

```diff
@@ -0,0 +1,888 @@
+                        tokenA: USDC_ADDRESS,
+                        tokenB: GHO_ADDRESS,
+                        smartCollateral: false,
+                        smartDebt: true,
+                        baseWithdrawalLimitInUSD: 0, // $0
+                        baseBorrowLimitInUSD: 10_000, // $10k
+                        maxBorrowLimitInUSD: 10_000 // $10k
+                    });
+                    setDexLimits(DEX_USDC_GHO); // Smart Debt
+
+                    DEX_FACTORY.setDexAuth(USDC_GHO_DEX, TEAM_MULTISIG, true);
+                }
+            }
+        }
+    }
+
+    // @notice Action 2: Set dust limits for sUSDe-GHO/USDC-GHO T4 vault
+    function action2() internal isActionSkippable(2) {
+        {
+            //sUSDe/GHO : USDC/GHO
+            address sUSDe_GHO_DEX_ADDRESS = getDexAddress(33);
+            address USDC_GHO_DEX_ADDRESS = getDexAddress(34);
+            address sUSDe_GHO__USDC_GHO_VAULT_ADDRESS = getVaultAddress(125);
+
+            {
+                // Update sUSDe-GHO<>USDC-GHO vault supply shares limit
+                IFluidAdminDex.UserSupplyConfig[]
+                    memory config_ = new IFluidAdminDex.UserSupplyConfig[](1);
+                config_[0] = IFluidAdminDex.UserSupplyConfig({
+                    user: sUSDe_GHO__USDC_GHO_VAULT_ADDRESS,
+                    expandPercent: 35 * 1e2, // 35%
+                    expandDuration: 6 hours, // 6 hours
+                    baseWithdrawalLimit: 5_000 * 1e18 // 5k shares ($10k)
+                });
+
+                IFluidDex(sUSDe_GHO_DEX_ADDRESS).updateUserSupplyConfigs(
+                    config_
+                );
+            }
+
+            {
+                // Update sUSDe-GHO<>USDC-GHO vault borrow shares limit
+                IFluidAdminDex.UserBorrowConfig[]
+                    memory config_ = new IFluidAdminDex.UserBorrowConfig[](1);
+                config_[0] = IFluidAdminDex.UserBorrowConfig({
+                    user: sUSDe_GHO__USDC_GHO_VAULT_ADDRESS,
+                    expandPercent: 30 * 1e2, // 30%
+                    expandDuration: 6 hours, // 6 hours
+                    baseDebtCeiling: 4_000 * 1e18, // 4k shares ($8k)
+                    maxDebtCeiling: 5_000 * 1e18 // 5k shares ($10k)
+                });
+
+                IFluidDex(USDC_GHO_DEX_ADDRESS).updateUserBorrowConfigs(
+                    config_
+                );
+            }
+
+            VAULT_FACTORY.setVaultAuth(
+                sUSDe_GHO__USDC_GHO_VAULT_ADDRESS,
+                TEAM_MULTISIG,
+                true
+            );
+        }
+    }
+
+    // @notice Action 3: Reduce allowance of fUSDC and fUSDT
+    function action3() internal isActionSkippable(3) {
+        address[] memory protocols = new address[](2);
+        address[] memory tokens = new address[](2);
+        uint256[] memory amounts = new uint256[](2);
+
+        {
+            /// fUSDC
+            uint256 allowance = IERC20(USDC_ADDRESS).allowance(
```

Comment:
no need to fetch, we can just set absolute amounts 

call to reserve contract at end of method is missing